utils/rpms/daos.spec

```diff
@@ -583,9 +583,18 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
-* Tue Feb 13 2024 Ashley M. Pittman <ashley.m.pittman@intel.com> 2.3.103-16
+* Thu Mar 21 2024 Ashley M. Pittman <ashley.m.pittman@intel.com> 2.5.101-3
```

Comment:
Worth updating the datestamp here (but only) if you have any other reason to push to this PR.